### ISSUE TITLE
Detail the Configuration Steps for Azure OpenAI Service

### ISSUE THREAD

**Issue by raghu4sangeeth** [MicrosoftDocs/power-platform#2322]
In Step 2 - Edit the connection to the Azure OpenAI Service.

Providing configuration settings of the Azure Open AI Service data source will be really helpful for the end users. This part is neither explained elsewhere nor available directly from the OpenAI studio. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ca242460-8478-cac4-676f-f0668989a024
* Version Independent ID: ca242460-8478-cac4-676f-f0668989a024
* Content: [Generative answers with Azure OpenAI on your data (preview) - Microsoft Copilot Studio](https://learn.microsoft.com/en-us/microsoft-copilot-studio/nlu-generative-answers-azure-openai)
* Content Source: [power-virtual-agents/nlu-generative-answers-azure-openai.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-virtual-agents/nlu-generative-answers-azure-openai.md)
* Service: **power-virtual-agents**
* GitHub Login: @mainguy70
* Microsoft Alias: **iawilt**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @raghu4sangeeth. This issue has been assigned to the doc author for review.

**Comment by KavehShahbazi** [comment]
Hi @raghu4sangeeth. Any lock finding this information? I'm trying to do the same thing with no luck so far. The Open AI model works fine on it's own but when I connect my copilot to it, it can't find the answer to user's questions

**Comment by louisfurhat** [comment]
Yes, I am having the same issue as well. Documentation on connection configuration is either lacking or there is another problem; it works in Openai Studio but not when deployed to Copilot Studio.

**Comment by sergejCampfireAI** [comment]
I am also experiencing this same issue as the rest not sure if anyone has been able to find how to resolve it?
